data/json/skills.json

```diff
@@ -1,16 +1,9 @@
 [
-  {
-    "type": "skill",
-    "id": "barter",
-    "name": { "str": "bartering" },
-    "description": "Your skill in bargaining, haggling, and trading with others.  Higher levels increase the odds of getting the better end of a deal, and might even see you convincing others to give you free stuff.",
-    "display_category": "display_social"
-  },
   {
     "type": "skill",
     "id": "speech",
-    "name": { "str": "speaking" },
-    "description": "Your skill in speaking to other people.  Covers ability in boasting, flattery, threats, persuasion, lies, and other facets of interpersonal communication.  Works best in conjunction with a high level of intelligence.",
+    "name": { "str": "social" },
+    "description": "Your skill in speaking to other people.  Covers ability in boasting, flattery, threats, persuasion, lies, bartering, and other facets of interpersonal communication.  Works best in conjunction with a high level of intelligence.",
     "companion_survival_rank_factor": 1,
     "display_category": "display_social",
```

Comment:
Just noticed. Since bartering is deprecated, social display category should go to. Otherwise you have a sole "social" skill in the "social" skill category. Bad UI.

Reply:
Could move Social to interacting. It is, after all, an interaction.